common-template/src/main/java/com/redhat/cloud/notifications/qute/templates/IntegrationType.java

```diff
@@ -0,0 +1,9 @@
+package com.redhat.cloud.notifications.qute.templates;
+
+public enum IntegrationType {
+    DRAWER,
+    EMAIL,
+    SLACK,
+    MS_TEAMS,
+    GOOGLE_CHAT
```

Comment:
Are these used?

Reply:
Not yes, it's more to give an idea about what is comming in couple weeks